frontend/src/app/main-table/volumes-table/volume-table.component.ts

```diff
@@ -1,21 +1,27 @@
 import {Component, Input, Output, EventEmitter} from "@angular/core";
 import {MatTableDataSource} from "@angular/material/table";
-import {Pvc} from "../../utils/types";
+import {Volume} from "../../utils/types";
+
+export type PvcWithStatus = {
+  pvc: Volume;
+  mountedBy: string | null;
+  status?: boolean;
```

Comment:
Putting status on this type would make sense if we were following the approach of the Notebooks table and the deletion status were coming from the API server. But here we are tracking deletion status purely in the front-end, and simply for UI display purposes.

So we want to keep ownership of that information inside the display component so that other components don't have to concern themselves with managing deletion state.

All that to say, let's keep deletion status tracking inside this component.

And, once you do, make sure you clean up status tracking for volumes that no longer exist. Otherwise it will be possible to recreate a deleted volume and have it incorrectly showing as deleting in the table once it starts getting listed by the API server.